src/docfx.website.themes/op.html/op.common.js

```diff
@@ -18,6 +18,14 @@ exports.getCanonicalUrl = function (canonicalUrlPrefix, path, layout) {
   return canonicalUrl;
 }
 
+exports.endsWith = function(str, suffix) {
```

Comment:
Move this function inside `exports.getCanonicalUrl`? `exports.xxx` is used to expose the function to other js.